### ISSUE TITLE
always pause at test

### ISSUE THREAD

**Issue by zongwan** [Homebrew/homebrew-nginx#173]
after
```brew update -v && brew upgrade -v ```
(last msg)
```
test -f '/usr/local/etc/nginx/mime.types' \
		|| cp conf/mime.types '/usr/local/etc/nginx'
cp conf/mime.types '/usr/local/etc/nginx/mime.types.default'
test -f '/usr/local/etc/nginx/fastcgi_params' \
		|| cp conf/fastcgi_params '/usr/local/etc/nginx'
cp conf/fastcgi_params \
		'/usr/local/etc/nginx/fastcgi_params.default'
test -f '/usr/local/etc/nginx/fastcgi.conf' \
		|| cp conf/fastcgi.conf '/usr/local/etc/nginx'
cp conf/fastcgi.conf '/usr/local/etc/nginx/fastcgi.conf.default'
test -f '/usr/local/etc/nginx/uwsgi_params' \
		|| cp conf/uwsgi_params '/usr/local/etc/nginx'
cp conf/uwsgi_params \
		'/usr/local/etc/nginx/uwsgi_params.default'
test -f '/usr/local/etc/nginx/scgi_params' \
		|| cp conf/scgi_params '/usr/local/etc/nginx'
cp conf/scgi_params \
		'/usr/local/etc/nginx/scgi_params.default'
test -f '/usr/local/etc/nginx/nginx.conf' \
		|| cp conf/nginx.conf '/usr/local/etc/nginx/nginx.conf'
cp conf/nginx.conf '/usr/local/etc/nginx/nginx.conf.default'
test -d '/usr/local/var/run' \
		|| mkdir -p '/usr/local/var/run'
test -d '/usr/local/var/log/nginx' \
		|| mkdir -p '/usr/local/var/log/nginx'
test -d '/usr/local/Cellar/nginx-full/1.9.14/html' \
		|| cp -R html '/usr/local/Cellar/nginx-full/1.9.14'
test -d '/usr/local/var/log/nginx' \
		|| mkdir -p '/usr/local/var/log/nginx'
```

**Comment by zongwan** [comment]
append gist-logs
https://gist.github.com/ec2d2ede5c117703aa4b7cff125f62b0

**Comment by denji** [comment]
This problem has been fixed in git 2.8.0+
git update 2.8.0+ & reload

https://github.com/Homebrew/homebrew-nginx/issues/167#issuecomment-203802511

**Comment by zongwan** [comment]
Thanks
